Tests/Editor/Deployment/DeploymentSettingsTests.cs

```diff
@@ -851,7 +869,7 @@ async Task Run()
                 DeploymentSettings underTest = GetUnitUnderTest(scenarioLocatorMock, coreApi: coreApiMock);
 
                 underTest.Refresh();
-                underTest.ScenarioIndex = 0;
+                underTest.Scenario = 0;
```

Comment:
```suggestion
                underTest.Scenario = DeploymentScenarios.SingleRegion
```

Reply:
There are a few other lines comparing to 0. Please modify those to the enum instead as well.

Reply:
Fixed!